datahub-web-react/src/app/entity/shared/tabs/Documentation/components/DescriptionEditor.tsx

```diff
@@ -27,14 +27,19 @@ export const DescriptionEditor = ({ onComplete }: { onComplete?: () => void }) =
         ? editedDescriptions[mutationUrn]
         : entityData?.editableProperties?.description || entityData?.properties?.description || '';
 
+    const editableName = entityData?.editableProperties?.name || '';
+
     const [updatedDescription, setUpdatedDescription] = useState(description);
     const [isDescriptionUpdated, setIsDescriptionUpdated] = useState(editedDescriptions.hasOwnProperty(mutationUrn));
     const [cancelModalVisible, setCancelModalVisible] = useState(false);
 
     const updateDescriptionLegacy = () => {
         const sanitizedDescription = DOMPurify.sanitize(updatedDescription);
         return updateEntity?.({
-            variables: { urn: mutationUrn, input: { editableProperties: { description: sanitizedDescription || '' } } },
+            variables: {
+                urn: mutationUrn,
+                input: { editableProperties: { name: editableName, description: sanitizedDescription || '' } },
```

Comment:
This will actually update the name to be an empty string (" ") if one was not previously set. This is not what we want .. Instead, we should be able to leave editable name null if there is not one set

Reply:
This makes sense to me and I will make the update.